zcash_primitives/CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this library adheres to Rust's notion of
 ### Added
 - `zcash_primitives::sapling::keys::DiversifiableFullViewingKey`
 - `zcash_primitives::sapling::keys::Scope`
+- `zcash_primitives::sapling::NullifierDerivingKey`
+- `zcash_primitives::sapling::DiversifiableFullViewingKey::to_nk`
```

Comment:
If this is the same `DiversifiableFullViewingKey` as was added above then we can simplify this by not mentioning `to_nk` as it's part of the same new type (I hadn't seen the changelog contents when making my prior comment, whereas now that there are other things mentioned here it's visible in the diff). Non-blocking, better to have more listed here than less, and we can clean this up before the next release.